src/ActivityRepository.js

```diff
@@ -13,11 +13,12 @@ class ActivityRepository {
     return todaysActivityRecord ? (todaysActivityRecord.minutesActive) : 0;
   }
 
-  getUsersMilesforDay(user, date) {
-    this.individualEntryRecords.filter(record => {
-      return record.date === date; 
-    })
-    return Math.round(this.getStepsForToday(date) * user.strideLength / 5280).toFixed(1); 
+  getUsersMilesForDay(user, date) {
+    if (this.getStepsForToday(date) > 0) {
+      return Number((this.getStepsForToday(date) * user.strideLength / 5280).toFixed(1)); 
+    } else {
+      return 0;
+    }
```

Comment:
Great use of a conditional and `Number` to ensure valid input and correct data type.